manifests/harvester.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: harvester-system
+---
+apiVersion: helm.cattle.io/v1
+kind: HelmChart
+metadata:
+  name: harvester
+  namespace: kube-system
+spec:
+  bootstrap: true
+  chart: https://localhost:9345/static/charts/$HARVESTER_CHART_TARBALL
+  targetNamespace: harvester-system
+  valuesContent: |-
+    auth:
+      enabled: true
+    containers:
+      apiserver:
+        image:
+          repository: rancher/harvester
+          tag: master-head
+          imagePullPolicy: "IfNotPresent"
+        hciMode: true
+    multus:
+      enabled: false
+    longhorn:
+      enabled: true
+      defaultSettings:
+        taintToleration: "kubevirt.io/drain:NoSchedule"
+    harvester-network-controller:
+      enabled: true
+      image:
+        pullPolicy: "IfNotPresent"
+    rancherEmbedded: true
+    webhook:
+      image:
+        repository: rancher/harvester-webhook
+        tag: master-head
+        imagePullPolicy: "IfNotPresent"
```

Comment:
can we use the set option like [previously](https://github.com/harvester/harvester-installer/blob/b0a0ebdaf2db53b3289d6c8624c7ff85ae180768/manifests/harvester.yaml#L15-L23) and do not include those default image values since they were defined by the chart we have packaged in.

Reply:
Is there any difference between the `valuesContent` and `set` ?

Reply:
Both will work for the same result, mostly we use `set` to overwrite simple default values and `valuesContent` for complex default values. Since both are working so the important stuff is we need to remove those image tags.

Reply:
I keep `imagePullPolicy: IfNotPresent` lines, they are mostly `Always` in the chart values file.